docs/self-hosting/configuration/domain-configuration.mdx

```diff
@@ -27,36 +27,165 @@ NEXTAUTH_URL=https://formbricks.example.com
 - Make sure your domain is properly configured in your DNS settings
 - If you're using HTTPS (recommended), ensure you have valid SSL certificates installed
 
-## Second Domain Setup (Survey Domain)
+## Public / Private Domain Split
 
-Formbricks allows you to serve surveys from a different domain than your main application. This is useful for:
+Formbricks now supports a powerful domain separation feature that allows you to serve public content (surveys, public APIs) from a different domain than your private admin interface. This provides enhanced security and better organization.
 
-- Separating your admin interface from public surveys
-- Using a dedicated domain for surveys
+### Why Use Domain Split?
+
+- **Enhanced Security**: Separate public-facing surveys from your admin interface
+- **Better Organization**: Use dedicated domains for different purposes (e.g., `admin.company.com` for management, `surveys.company.com` for public surveys)
+- **Improved Performance**: Optimize each domain for its specific purpose
```

Comment:
I don't think improved performance can be achieved with the domain split. I would suggest to remove this point.